### DISCUSSION TITLE
Problem renewing my github pro student benefits

### DISCUSSION THREAD

**Discussion by alcantaragiubs** [community/community#61395] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

Hi, my name is Giulia and I'm having a problem while trying to renew my github pro student benefits.
I'm completing all the informations that are requested, but even though the system doesn't allow me to click on the "Continue" option and get my benefits back. 
I used to enjoy all the benefits I could in my studying journey in my University period during this firt half of the year, so I'd thank you very much if I could solve this situation as soon as possible.
Im gonna send you the img of the problem for you to understand.
Thanks for the attention and have nice day.

![image](https://github.com/community/community/assets/54143767/ec7b7798-d58e-4f8f-9887-46e14081be13)

**Comment by mdnurulmomen** [comment]
In order to get the GitHub Pro badge, you need to subscribe to the GitHub Pro plan.

See which options below best apply to your situation:

If you are still a student with a valid school email address, apply for the [GitHub Student Developer pack](https://education.github.com/pack) - which provides you free access to GitHub Pro (while you remain a student).
If you are no longer a student, you can subscribe to GitHub Pro on your own which should cost $4 USD monthly.
For steps on how to subscribe to GitHub Pro on your own, you can refer to: [GitHub Documentation - Step by Step instructions on how to subscribe to GitHub Pro plan](https://docs.github.com/en/billing/managing-the-plan-for-your-github-account/upgrading-your-accounts-plan#upgrading-your-personal-accounts-plan)

Additional reference: [Benefits of GitHub Pro plan](https://docs.github.com/en/get-started/learning-about-github/githubs-plans#github-pro)

Hope this answers & resolves your question.
Take care and have a good day ahead !
